Hush/Assets/Plugins/PyroParticles/Prefab/Script/FireCollisionForwardScript.cs

```diff
@@ -19,13 +17,13 @@ public class FireCollisionForwardScript : MonoBehaviour
     {
         public ICollisionHandler CollisionHandler;
 
-        public void OnCollisionEnter(Collision col)
+        public virtual void OnCollisionEnter(Collision col)
+        {
+            HandleCollision(col);
+        }
+
+        protected virtual void HandleCollision(Collision col)
         {
-            if (col.gameObject.CompareTag(Tags.Player) || col.gameObject.CompareTag(Tags.Ground) )
```

Comment:
Moved to a custom class that inherits from this scripts instead of modifying the original one... better not to modify 3rd party code if we can avoid doing so! The class is in the other PR i think

Reply:
oh, I see that makes sense. I wasn't seeing the custom class so I was just confused